<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cv</class>
 <widget class="QMainWindow" name="Cv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel {

    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    border-radius: 10px;
    color: white;
    font-size: 11px;
    padding: 8px 12px;
}</string>
    </property>
    <property name="text">
     <string>hello ki</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRollNumber">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel {

    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    border-radius: 10px;
    color: white;
    font-size: 11px;
    padding: 8px 12px;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDepartment">
    <property name="geometry">
     <rect>
      <x>542</x>
      <y>50</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel {

    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    border-radius: 10px;
    color: white;
    font-size: 11px;
    padding: 8px 12px;
}</string>
    </property>
    <property name="text">
     <string>DEPARTMENT</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSemester">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel {

    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    border-radius: 10px;
    color: white;
    font-size: 11px;
    padding: 8px 12px;
}</string>
    </property>
    <property name="text">
     <string>SEMESTER</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="skillsTable">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>581</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ROLL NUMBER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DEPARTMENT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>40</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SEMESTER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    qproperty-alignment: 'AlignCenter'; /* Center-aligns the text */
f;
    font-size: 30px;
}</string>
    </property>
    <property name="text">
     <string>SKILLS</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditDescription">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>771</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* QTextEdit */
QTextEdit {
    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    border-radius: 10px;
    color: white;
    font-size: 14px;
    padding: 10px;
    selection-background-color: #ff5c93;
    selection-color: #121212;
}

/* Scroll Bar */
QScrollBar:vertical {
    background: #1f1f1f;
    width: 12px;
    margin: 4px;
    border-radius: 6px;
}

QScrollBar::handle:vertical {
    background: #ff5c93;
    border-radius: 6px;
    min-height: 20px;
}

QScrollBar::add-line, QScrollBar::sub-line {
    background: none;
}

/* QTextEdit Placeholder */
QTextEdit[placeholderText] {
    color: #a0a0a0;
    font-style: italic;
}
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
